### PR TITLE
Added endpoint for TimeSeries

### PR DESCRIPTION
**Pull request by kosmydel**
Prepared endpoints for the time series view:
- `/api/timeseries/locations` - it returns the list of sensors,
- `/api/timeseries/data` - it returns the date for the chart.

Example call of `data` endpoint:
`http://localhost:8080/api/timeseries/data?dateFrom=2021-01-01&dateTo=2021-01-02&pollutions=PM25,PM10&sensors=MpKrakBujaka`

### CODE REVIEW

**Comment by kosmydel** [comment]
New call to the API of the pollution data:
```
http://localhost:8080/api/timeseries/data?dateFrom=2000-01-01&dateTo=2023-06-02&sensors=MpKrakBujaka&pollution1=PM25&pollution2=PM10
```

File: src/main/java/atmosfears/AtmosFearsBE/controller/DataIngestController.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public ResponseEntity<String> loadAndPreprocessData() {
     Set<String> dataFiles = dataLoadService.collectAndSaveData();
     for(String file : dataFiles) {
       AirParticulatesFile airParticulatesFile = calculationService.getPredictionsForFile(file);
-      airParticulatesService.saveFile(airParticulatesFile);
+      //airParticulatesService.saveFile(airParticulatesFile);
```
Comment (kacperklusek) [inline src/main/java/atmosfears/AtmosFearsBE/controller/DataIngestController.java]: remove commented lines

File: src/main/java/atmosfears/AtmosFearsBE/controller/HelloController.java [hunk 2]
```diff
@@ -15,7 +17,8 @@ public HelloController(AirParticulatesService airParticulatesService) {
 
     @GetMapping("/")
     public String index() {
-        return "Greetings from Spring Boot! <br> Repository findAll result: " + airParticulatesService.findAll();
+        return "Greetings from Spring Boot! <br> Repository getAverage result: " +
+                airParticulatesService.getAverageParticulatesValues(new Date("01/01/2021 00:00:00"), new Date("01/01/2021 04:00:00"));
```
Comment (kacperklusek) [inline src/main/java/atmosfears/AtmosFearsBE/controller/HelloController.java]: i believe we don't need this whole file.

File: src/main/java/atmosfears/AtmosFearsBE/database/Particulate.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+public enum Particulate {
+
+    // TODO: Add correct units for each particulate
+    CO("mg/m3"),
+    NO2("µg/m3"),
+    O3("µg/m3"),
+    PM10("µg/m3"),
+    PM25("µg/m3"),
+    SO2("µg/m3");
```
Comment (kacperklusek) [inline src/main/java/atmosfears/AtmosFearsBE/database/Particulate.java:13]: I believe we don't have these units? or did you forget about this TODO?

File: src/main/java/atmosfears/AtmosFearsBE/model/AirParticulatesFile.java [hunk 4]
```diff
@@ -7,7 +7,8 @@
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 
-@Document(collection = "air_particulates")
+//TODO  DELETE THIS
+//@Document(collection = "air_particulates")
```
Comment (kacperklusek) [inline src/main/java/atmosfears/AtmosFearsBE/model/AirParticulatesFile.java]: Delete this

**Review by kacperklusek — CHANGES_REQUESTED** [review]

File: src/main/java/atmosfears/AtmosFearsBE/database/Particulate.java (hunk 3 above)
Comment (kosmydel) [reply src/main/java/atmosfears/AtmosFearsBE/database/Particulate.java:13]: I use it in `TimeSeriesController`. This unit is displayed on the `TimeSeries` chart.

**Review by kacperklusek — APPROVED** [review]

**Pull request merged**